plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -363,7 +371,10 @@ export function wrapFieldResolver<TSource = any, TContext = any, TArgs = any>(
 
     let field: any;
     let shouldEndSpan = false;
-    if (config.depth >= 0 && config.depth < depth) {
+    if (
+      (config.depth >= 0 && config.depth < depth) ||
+      ignoreMap().has(path[path.length - 1])
```

Comment:
What is the motivation of running the resolver with the context of parent span, instead of just not instrumenting this call?

If I understand correctly, then for this query
```
{
  foo {
    bar
  }
}
```

if bar is ignored, then downstream spans created under the `bar` resolver will be children of `foo`, right?

Is this what we want?